diff -up powerdevil-5.18.90/daemon/powerdevilprofilegenerator.cpp.1~ powerdevil-5.18.90/daemon/powerdevilprofilegenerator.cpp
--- powerdevil-5.18.90/daemon/powerdevilprofilegenerator.cpp.1~	2020-05-14 12:56:45.000000000 +0200
+++ powerdevil-5.18.90/daemon/powerdevilprofilegenerator.cpp	2020-05-16 00:11:31.916603161 +0200
@@ -61,13 +61,13 @@ void ProfileGenerator::generateProfiles(
     // We want to dim the screen after a while, definitely
     {
         KConfigGroup dimDisplay(&acProfile, "DimDisplay");
-        dimDisplay.writeEntry< int >("idleTime", 300000);
+        dimDisplay.writeEntry< int >("idleTime", 3600000);
     }
 
     auto initLid = [toRam](KConfigGroup &profile)
     {
         const bool mobile = !qEnvironmentVariableIsEmpty("QT_QUICK_CONTROLS_MOBILE");
-        const Modes defaultPowerButtonAction = mobile ? ToRamMode : LogoutDialogMode;
+        const Modes defaultPowerButtonAction = mobile ? ToRamMode : ToRamMode;
 
         KConfigGroup handleButtonEvents(&profile, "HandleButtonEvents");
         handleButtonEvents.writeEntry< uint >("powerButtonAction", defaultPowerButtonAction);
@@ -85,7 +85,7 @@ void ProfileGenerator::generateProfiles(
     // And we also want to turn off the screen after another while
     {
         KConfigGroup dpmsControl(&acProfile, "DPMSControl");
-        dpmsControl.writeEntry< uint >("idleTime", 600);
+        dpmsControl.writeEntry< uint >("idleTime", 5400);
     }
 
     // Powersave
