diff -up powerdevil-master/CMakeLists.txt.1~ powerdevil-master/CMakeLists.txt
--- powerdevil-master/CMakeLists.txt.1~	2023-09-09 04:15:48.000000000 +0200
+++ powerdevil-master/CMakeLists.txt	2023-09-09 16:41:28.503893170 +0200
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.16)
 
 project(PowerDevil)
 set(PROJECT_VERSION "5.27.80")
-set(PROJECT_VERSION_MAJOR 5)
+set(PROJECT_VERSION_MAJOR 6)
 
 set(QT_MIN_VERSION "6.4.0")
 set(KF6_MIN_VERSION "5.240.0")
diff -up powerdevil-master/daemon/CMakeLists.txt.1~ powerdevil-master/daemon/CMakeLists.txt
--- powerdevil-master/daemon/CMakeLists.txt.1~	2023-09-09 16:41:28.503893170 +0200
+++ powerdevil-master/daemon/CMakeLists.txt	2023-09-09 16:42:48.657650357 +0200
@@ -49,7 +49,7 @@ qt_add_dbus_interface(powerdevilcore_SRC
 )
 
 add_library(powerdevilcore SHARED ${powerdevilcore_SRCS} powerdevil_debug.cpp)
-set_target_properties(powerdevilcore PROPERTIES VERSION ${PROJECT_VERSION} SOVERSION 2)
+set_target_properties(powerdevilcore PROPERTIES VERSION ${PROJECT_VERSION} SOVERSION 3)
 
 generate_export_header(powerdevilcore)
 
