--- powerdevil-5.7.0/daemon/BackendConfig.cmake.0000~	2016-07-04 17:24:50.672120782 +0200
+++ powerdevil-5.7.0/daemon/BackendConfig.cmake	2016-07-04 18:24:18.141079509 +0200
@@ -18,6 +18,8 @@ set(powerdevilupowerbackend_SRCS
     backends/upower/udevqtdevice.cpp
 )
 
+add_definitions(-DHAVE_UDEV)
+
 set_source_files_properties(
 ${CMAKE_CURRENT_SOURCE_DIR}/backends/upower/dbus/org.freedesktop.UPower.xml
 ${CMAKE_CURRENT_SOURCE_DIR}/backends/upower/dbus/org.freedesktop.UPower.Device.xml
