diff -up powerdevil-5.19.90/daemon/powerdevilprofilegenerator.cpp.1~ powerdevil-5.19.90/daemon/powerdevilprofilegenerator.cpp
--- powerdevil-5.19.90/daemon/powerdevilprofilegenerator.cpp.1~	2020-09-17 20:05:01.303990953 +0200
+++ powerdevil-5.19.90/daemon/powerdevilprofilegenerator.cpp	2020-09-17 20:08:53.131487621 +0200
@@ -63,7 +63,7 @@ void ProfileGenerator::generateProfiles(
     // We want to dim the screen after a while, definitely
     {
         KConfigGroup dimDisplay(&acProfile, "DimDisplay");
-        dimDisplay.writeEntry< int >("idleTime", 300000);
+        dimDisplay.writeEntry< int >("idleTime", 3600000);
     }
 
     auto initLid = [toRam, mobile](KConfigGroup &profile)
@@ -108,8 +108,8 @@ void ProfileGenerator::generateProfiles(
 
     // We want to turn off the screen after another while
     {
-        // on mobile, 1 minute, on laptop 5 minutes
-        auto timeout = mobile ? 60 : 300;
+        // on mobile, 1 minute, on laptop 90 minutes
+        auto timeout = mobile ? 60 : 5400;
         KConfigGroup dpmsControl(&batteryProfile, "DPMSControl");
         dpmsControl.writeEntry< uint >("idleTime", timeout);
         dpmsControl.writeEntry< uint >("lockBeforeTurnOff", mobile);
